Makefile

```diff
@@ -312,16 +312,16 @@ delete_all_virsh_resources: destroy_all_nodes delete_minikube kill_all_port_forw
 	skipper run $(SKIPPER_PARAMS) 'discovery-infra/delete_nodes.py -ns $(NAMESPACE) -a'
 
 _download_logs:
-	discovery-infra/download_logs.py $(REMOTE_SERVICE_URL) $(LOGS_DEST) --cluster-id $(CLUSTER_ID)
+	discovery-infra/download_logs.py $(REMOTE_SERVICE_URL) $(LOGS_DEST) --cluster-id $(CLUSTER_ID) $(ADDITIONAL_PARAMS)
 
 download_logs:
-	skipper make $(SKIPPER_PARAMS) _download_logs
+	skipper make $(SKIPPER_PARAMS) _download_logs $(ADDITIONAL_PARAMS)
 
 _download_all_logs:
```

Comment:
You can delete `_download_all_logs` and call from `download_all_logs` to `_download_logs` directly.